libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts

```diff
@@ -339,10 +340,10 @@ export const ParentalLeaveForm: Form = buildForm({
             buildRadioField({
               id: 'usePersonalAllowanceFromSpouse',
               title: parentalLeaveFormMessages.personalAllowance.useFromSpouse,
-              condition: (answers) => {
-                // TODO add check if this person has a spouse...
-                return true
-              },
+              condition: (answers) =>
+                answers.otherParent === SPOUSE ||
+                (answers.otherParent === MANUAL &&
+                  answers.otherParentRightOfAccess === YES),
```

Comment:
We'll need to give `usePersonalAllowanceFromSpouse` a default value in the cases where it is skipped, since undefined will [result in an error when transforming/serializing the application](https://github.com/island-is/island.is/blob/main/libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.utils.ts#L23) to API payload, or refactor the [transformation/serialization code](https://github.com/island-is/island.is/blob/main/libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.utils.ts#L47).

Reply:
Good catch, I will fix that, thanks 👍